README.rst

```diff
@@ -195,6 +195,18 @@ I also accept pull requests on this, if they're small, atomic, and if they make
     :target: https://github.com/Ouranosinc/cookiecutter-pypackage/actions/workflows/main.yml
     :alt: Build Status
 
+.. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
+        :target: https://github.com/psf/black
+        :alt: Python Black
```

Comment:
I thought that `ruff` effectively replaced `black`?

Reply:
Essentially. I could remove it in the next update.